priority_alerts/views.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework import (
+    permissions, generics
+)
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework_gis import filters as gis_filters
+from priority_alerts import (
+    serializers,
+    models,
+    filters as alerts_filters
+)
+
+
+class AuthModelMixIn:
+    """Default Authentication for priority_alerts views."""
+    permission_class = (permissions.AllowAny,)
+
+
+class AlertsView(generics.ListAPIView):
+    """View..."""
```

Comment:
docstring must say what it will do... not temporarily.

Reply:
Adjusted.